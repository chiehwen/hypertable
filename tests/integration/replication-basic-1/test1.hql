CREATE NAMESPACE "/rep";
CREATE NAMESPACE "/source";
USE "/source";
CREATE TABLE NoRepl (c);
INSERT INTO NoRepl VALUES ("r00", "c", "v00");
INSERT INTO NoRepl VALUES ("r01", "c", "v01");
INSERT INTO NoRepl VALUES ("r02", "c", "v02");
INSERT INTO NoRepl VALUES ("r03", "c", "v03");
INSERT INTO NoRepl VALUES ("r04", "c", "v04");
INSERT INTO NoRepl VALUES ("r05", "c", "v05");
INSERT INTO NoRepl VALUES ("r06", "c", "v06");
INSERT INTO NoRepl VALUES ("r07", "c", "v07");
INSERT INTO NoRepl VALUES ("r08", "c", "v08");
INSERT INTO NoRepl VALUES ("r09", "c", "v09");
INSERT INTO NoRepl VALUES ("r10", "c", "v10");
INSERT INTO NoRepl VALUES ("r11", "c", "v11");
INSERT INTO NoRepl VALUES ("r12", "c", "v12");
INSERT INTO NoRepl VALUES ("r13", "c", "v13");
INSERT INTO NoRepl VALUES ("r14", "c", "v14");
INSERT INTO NoRepl VALUES ("r15", "c", "v15");
INSERT INTO NoRepl VALUES ("r16", "c", "v16");
INSERT INTO NoRepl VALUES ("r17", "c", "v17");
INSERT INTO NoRepl VALUES ("r18", "c", "v18");
INSERT INTO NoRepl VALUES ("r19", "c", "v19");
INSERT INTO NoRepl VALUES ("r20", "c", "v20");
INSERT INTO NoRepl VALUES ("r21", "c", "v21");
INSERT INTO NoRepl VALUES ("r22", "c", "v22");
INSERT INTO NoRepl VALUES ("r23", "c", "v23");
INSERT INTO NoRepl VALUES ("r24", "c", "v24");
INSERT INTO NoRepl VALUES ("r25", "c", "v25");
INSERT INTO NoRepl VALUES ("r26", "c", "v26");
INSERT INTO NoRepl VALUES ("r27", "c", "v27");
INSERT INTO NoRepl VALUES ("r28", "c", "v28");
INSERT INTO NoRepl VALUES ("r29", "c", "v29");

CREATE TABLE Repl (c) REPLICATE TO "cluster";
INSERT INTO Repl VALUES ("r00", "c", "value00");
INSERT INTO Repl VALUES ("r01", "c", "value01");
INSERT INTO Repl VALUES ("r02", "c", "value02");
INSERT INTO Repl VALUES ("r03", "c", "value03");
INSERT INTO Repl VALUES ("r04", "c", "value04");
INSERT INTO Repl VALUES ("r05", "c", "value05");
INSERT INTO Repl VALUES ("r06", "c", "value06");
INSERT INTO Repl VALUES ("r07", "c", "value07");
INSERT INTO Repl VALUES ("r08", "c", "value08");
INSERT INTO Repl VALUES ("r09", "c", "value09");
INSERT INTO Repl VALUES ("r10", "c", "value10");
INSERT INTO Repl VALUES ("r11", "c", "value11");
INSERT INTO Repl VALUES ("r12", "c", "value12");
INSERT INTO Repl VALUES ("r13", "c", "value13");
INSERT INTO Repl VALUES ("r14", "c", "value14");
INSERT INTO Repl VALUES ("r15", "c", "value15");
INSERT INTO Repl VALUES ("r16", "c", "value16");
INSERT INTO Repl VALUES ("r17", "c", "value17");
INSERT INTO Repl VALUES ("r18", "c", "value18");
INSERT INTO Repl VALUES ("r19", "c", "value19");
INSERT INTO Repl VALUES ("r20", "c", "value20");
INSERT INTO Repl VALUES ("r21", "c", "value21");
INSERT INTO Repl VALUES ("r22", "c", "value22");
INSERT INTO Repl VALUES ("r23", "c", "value23");
INSERT INTO Repl VALUES ("r24", "c", "value24");
INSERT INTO Repl VALUES ("r25", "c", "value25");
INSERT INTO Repl VALUES ("r26", "c", "value26");
INSERT INTO Repl VALUES ("r27", "c", "value27");
INSERT INTO Repl VALUES ("r28", "c", "value28");
INSERT INTO Repl VALUES ("r29", "c", "value29");
DELETE * FROM Repl WHERE ROW = "r28";

INSERT INTO NoRepl VALUES ("r00", "c", "v00");
INSERT INTO NoRepl VALUES ("r01", "c", "v01");
INSERT INTO NoRepl VALUES ("r02", "c", "v02");
INSERT INTO NoRepl VALUES ("r03", "c", "v03");
