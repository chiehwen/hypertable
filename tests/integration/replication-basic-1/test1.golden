
Welcome to the hypertable command interpreter.
For information about Hypertable, visit http://hypertable.com

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

USE "/rep";
GET LISTING;
source	(namespace)
USE "source";
GET LISTING;
Repl
SHOW CREATE TABLE Repl;

CREATE TABLE Repl (
  c,
  ACCESS GROUP default (c)
)

SELECT * FROM Repl;
r00	c	value00
r01	c	value01
r02	c	value02
r03	c	value03
r04	c	value04
r05	c	value05
r06	c	value06
r07	c	value07
r08	c	value08
r09	c	value09
r10	c	value10
r11	c	value11
r12	c	value12
r13	c	value13
r14	c	value14
r15	c	value15
r16	c	value16
r17	c	value17
r18	c	value18
r19	c	value19
r20	c	value20
r21	c	value21
r22	c	value22
r23	c	value23
r24	c	value24
r25	c	value25
r26	c	value26
r27	c	value27
r29	c	value29
